integrations/atlassian/confluence/client.go

```diff
@@ -75,3 +76,33 @@ func connStatus(i *integration) sdkintegrations.OptFn {
 		}
 	})
 }
+
+// connTest is an optional connection test provided by the integration
+// to AutoKitteh. It is used to verify that the connection is working
+// as expected. The possible results are "OK" and "error".
+func connTest(i *integration) sdkintegrations.OptFn {
+	return sdkintegrations.WithConnectionTest(func(ctx context.Context, cid sdktypes.ConnectionID) (sdktypes.Status, error) {
+		if !cid.IsValid() {
+			return sdktypes.NewStatus(sdktypes.StatusCodeWarning, "init required"), nil
```

Comment:
Please capitalize status messages in this PR.
I'll create a quick PR to do it in all existing `connStatus` functions.

Reply:
Done